webapp/src/css/private-vars.less

```diff
@@ -0,0 +1,29 @@
+/**
+* WARNING:
+* Don't import this file. This is only for "variable.less" usage.
+* Don't use these variables. The variable names aren't descriptive enough.
+* Instead, use the variables from "variable.less".
+*/
+@white: #FFFFFF;
+@gray-ultra-light: #F2F2F2;
+@gray-light: #E0E0E0;
+@gray-medium: #A0A0A0;
+@gray-dark: #777777;
+@gray-ultra-dark: #333333;
+@black: #000000; ///aaaaaaa
```

Comment:
Is the comment still needed?

Reply:
hahaha sorry for the weird comment, I forgot to remove it. Thanks for catching it